src/components/relaying-education-series/index.tsx

```diff
@@ -0,0 +1,43 @@
+import WhatIsRelaying from '@/components/relaying-education-series/steps/WhatIsRelaying'
+import HowItWorks from '@/components/relaying-education-series/steps/HowItWorks'
+import Benefits from '@/components/relaying-education-series/steps/Benefits'
+import TechnicalInformation from '@/components/relaying-education-series/steps/TechnicalInformation'
+import RelaySeriesStepper from '@/components/relaying-education-series/RelaySeriesStepper'
+import { type EducationSeriesStepperProps } from '@/components/relaying-education-series/RelaySeriesStepper/useEducationSeriesStepper'
+import { useRouter } from 'next/router'
+import { AppRoutes } from '@/config/routes'
+import { type UrlObject } from 'url'
+
+const RelayingSeriesSteps: EducationSeriesStepperProps['steps'] = [
+  {
+    title: 'What is relaying?',
+    render: (_, onNext, onClose) => <WhatIsRelaying onNext={onNext} onClose={onClose} />,
+  },
+  {
+    title: 'How it works?',
+    render: (onBack, onNext, onClose) => <HowItWorks onBack={onBack} onNext={onNext} onClose={onClose} />,
+  },
+  {
+    title: 'Benefits',
+    render: (onBack, onNext, onClose) => <Benefits onBack={onBack} onNext={onNext} onClose={onClose} />,
+  },
+  {
+    title: 'Technical information',
+    render: (onBack, _, onClose) => <TechnicalInformation onBack={onBack} onClose={onClose} />,
+  },
+]
```

Comment:
It might help you to pass these steps to a stepper creator outside of the component, then you can use `useEducationSeriesStepper` everywhere without having to pass the props each time to the `render`. ([An example](https://github.com/safe-global/safe-dao-governance-app/blob/dev/src/services/StepperFactory.tsx#L10) of this can be found in the Governance App.)

Reply:
I initially tried to use that stepper creator but then I've decided to use the same pattern as in `CardStepper` and `TxStepper`  that are what we have in this repo.